tests/integration-tests/tests/common/utils.py

```diff
@@ -231,3 +235,33 @@ def get_route_tables(subnet_id, region):
         Filters=[{"Name": "association.subnet-id", "Values": [subnet_id]}]
     )
     return [table["RouteTableId"] for table in response["RouteTables"]]
+
+
+def gather_system_report(cluster, request, scheduler_commands, remote_command_executor):
+    out_dir = request.config.getoption("output_dir")
+    logging.info(f"Get system information about test: {request.node.name}")
+    logging.info(f"Run command on head node to get system information about test: {request.node.name}")
```

Comment:
You can just write: "Retrieve head node system information for test: ...."

Reply:
Done